redash/query_runner/athena.py

```diff
@@ -149,7 +149,12 @@ def run_query(self, query, user):
             column_tuples = [(i[0], _TYPE_MAPPINGS.get(i[1], None)) for i in cursor.description]
             columns = self.fetch_columns(column_tuples)
             rows = [dict(zip(([c['name'] for c in columns]), r)) for i, r in enumerate(cursor.fetchall())]
-            data = {'columns': columns, 'rows': rows}
+            qbytes = None
+            try:
+                qbytes = cursor.data_scanned_in_bytes()
+            except AttributeError as e:
+                debug("Athena Direct can't get data_scanned_in_bytes: %s", e)
+            data = { 'columns': columns, 'rows': rows, 'data_scanned': [{ qbytes }] }
```

Comment:
I quickly scanned the Athena AWS docs. I didn't see this returned from normal queries. It looks like you need to request the query execution plan and I don't see that happening here. But maybe there's some abstraction I'm not seeing.